frontend/src/components/stages/transfer_view.ts

```diff
@@ -48,36 +54,57 @@ export class TransferView extends MobxLitElement {
 
     return html`
       <stage-description .stage=${this.stage}></stage-description>
-      ${this.renderCountdown()}
-      ${this.renderOverlay()}
+
+      <div>
+        Please wait while we transfer you to the next stage of the experiment.
```

Comment:
Does it make sense to not hardcode text here since experimenters can customize it via the stage description?